multimedia/minisatip/Makefile

```diff
@@ -0,0 +1,36 @@
+#
+# This is free software, licensed under the GNU General Public License v2.
+# See /LICENSE for more information.
+#
+
+include $(TOPDIR)/rules.mk
+
+PKG_NAME:=minisatip
+PKG_VERSION:=0.7.15
+PKG_RELEASE:=1
+
+PKG_SOURCE_PROTO := git
+PKG_SOURCE_URL := https://github.com/catalinii/minisatip.git
+PKG_SOURCE_VERSION := fdbf44fc48392b0735e0bf17b31aaff8f06740d0
```

Comment:
I cannot find the `PKG_VERSION:=0.7.15` in the GitHub repo.  How does it correlate with the source version?  If it's just a random commit id, please consider using commit date of the format yyyy-mm-dd as the version.

And there is no need for spaces around `:=`, they should be stripped out.